### ISSUE TITLE
[bug] `client.moveToActive is not a function` when using the Redis queue driver

### ISSUE THREAD

**Issue by elbojoloco** [soketi/soketi#205]
My `.env`:
```
APP_MANAGER_DRIVER=mysql

DB_MYSQL_USERNAME=soketi
DB_MYSQL_PASSWORD=soketi
DB_MYSQL_DATABASE=soketi

SOKETI_DEBUG=1

QUEUE_DRIVER=redis
```

soketi version: `@soketi/soketi@0.20.0`
redis version: `6.2.1`
node version: `v16.13.1`

I run `soketi start` in the directory of the above `.env` and this is my output (stuck in infinite loop it seems):
![image](https://user-images.githubusercontent.com/7912315/146202682-04dfbda5-e154-4730-a03f-06a5b42157ae.png)

Everything works fine when disabling the redis queue driver. Any help with this?

**Comment by rennokki** [comment]
Are you using Redis in cluster mode?

**Comment by elbojoloco** [comment]
Not that I know of, I'm using [dbngin](https://dbngin.com/) to run a local redis server without any additional configs.

**Comment by costafacchini** [comment]
I have the same problem in my app.
This error started to appear in the update to version 1.57.4

**Comment by elbojoloco** [comment]
@costafacchini version 1.57.4 of what exactly?

**Comment by costafacchini** [comment]
@elbojoloco 
I want to apologize for the comment.
I have this same error, however the problem is not in another lib.
Please disregard my previous comment.

**Comment by rennokki** [comment]
@costafacchini is right. There is a BullMQ PR that was merged and created this issue: https://github.com/taskforcesh/bullmq/pull/933

The best I can do for now is to downgrade to v1.57.3

**Comment by rennokki** [comment]
@costafacchini @elbojoloco Please let me know if this issue was fixed with the latest version.

**Comment by costafacchini** [comment]
@rennokki Yes, the issue was fixed in the last version (1.54.9)

**Comment by rennokki** [comment]
@costafacchini Good, I will update bullmq only if they fixed this specific issue.

**Comment by rennokki** [comment]
They seem to have it fixed on 1.59.3 😁
